contracts/ColonyNetworkStorage.sol

```diff
@@ -73,4 +73,16 @@ contract ColonyNetworkStorage is DSAuth, DSMath {
 
   // Mapping containing the last auction start timestamp for a token address
   mapping (address => uint) recentAuctions;
+
+  // Address of the ENS registrar for joincolony.eth
+  address ens;
+  // Namehash of the root node that we administer (i.e. namehash("joincolony.eth"))
+  bytes32 rootNode;
+  // Namehash of the users node that we administer (i.e. namehash("users.joincolony.eth"))
```

Comment:
`users` is plural, but `colony` is singular.... This is the sort of thing that would annoy me down the line. I think I prefer `users`->`user`, unless someone has a good argument to the contrary?

Reply:
Or put `colonies` in plural?

Reply:
After a moments reflection I also prefer the singular.